vendor/github.com/containers/common/libnetwork/pasta/types.go

```diff
@@ -10,6 +10,9 @@ type SetupResult struct {
 	// DNSForwardIP is the ip used in --dns-forward, it should be added as first
 	// entry to resolv.conf in the container.
 	DNSForwardIPs []string
+	// MapGuestIps are the ips used for the --map-guest-addr option which
+	// we can use for the host.containers.internal entry.
+	MapGuestAddrIPs []string
```

Comment:
Note that this can have at most one IPv4 and one IPv6 address (for now, anyway).